### PR TITLE
adding wizard component <?.x.?>

### PR DESCRIPTION
**Pull request by ryelo**

### CODE REVIEW

File: src/PresentationalComponents/Wizard/Wizard.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        let renderModalActions =  [
+            <Button key="cancel" variant="secondary" onClick={ this.handleOnClose }>
+            Cancel
+            </Button>,
+            // Conditionally render 'previous' button if not on first page
+            this.state.currentStep > 0
+                ? <Button key="previous" variant="secondary" onClick={ this.handlePreviousModalStep }> Previous </Button>
+                : null,
+            // Conditionally render 'confirm' button if on last page
+            this.state.currentStep < this.props.steps - 1
```
Comment (karelhala) [inline src/PresentationalComponents/Wizard/Wizard.js]: No need to pass the steps to wizard but use content array.
```JSX
this.state.currentStep < this.props.content.length - 1
```

File: src/PresentationalComponents/Wizard/Wizard.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            // eslint-disable-next-line
+            console.error(`[WIZARD] You specified ${this.props.steps} steps, but only passed content for ${this.props.content.length} steps`);
+        }
+
+        let renderModalActions =  [
+            <Button key="cancel" variant="secondary" onClick={ this.handleOnClose }>
+            Cancel
+            </Button>,
+            // Conditionally render 'previous' button if not on first page
+            this.state.currentStep > 0
```
Comment (karelhala) [inline src/PresentationalComponents/Wizard/Wizard.js]: It's better to use JS shortcut here
```JSX
this.state.currentStep !== 0 && 
  <Button key="previous" variant="secondary" onClick={ this.handlePreviousModalStep }> Previous </Button>
```

**Comment by ryelo** [comment]
@karelhala mind looking at this again?

**Pull request merged**